### PR TITLE
Changes to run with 0.1-alpha

### PR DESCRIPTION
**Pull request by mrtom**
Using the `ent-starter` template today fails when running `npm run codegen` with a TS error.

Honestly I didn't look into it much (and can do if useful), but instead upgraded everything to the 0.1-alpha version. After this problem the issue goes away.

Given the [docs](https://ent.dev/docs/intro) are using the 0.1 format already I figure this project should just be updated to match.

I also added a `.nvmrc` file for Node 17

Test Plan:
Created a new repo using my fork as a template, went through the `README` and confirmed I could spin up a GQL server correctly.

### CODE REVIEW

**Comment by lolopinto** [comment]
thanks for doing this. has been on my list but i somehow hadn't done it for some reason

i even have https://github.com/lolopinto/ent-starter/tree/v0.1 which has some of this but not all of it

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,14 @@
-FROM ghcr.io/lolopinto/ent:0.0.33-nodejs-17-slim
+FROM ghcr.io/lolopinto/ent:0.1.0-alpha.5-nodejs-17-dev
```
Comment (lolopinto) [inline Dockerfile:1]: can you update to `ghcr.io/lolopinto/ent:0.1.0-alpha.30-nodejs-17-slim`

that's the latest

File: README.md [hunk 2]
```diff
@@ -35,14 +35,15 @@ Example schema change:
 ```ts
 // src/schema/user.ts
```
Comment (lolopinto) [inline README.md:36]: `user_schema.ts`

File: README.md [hunk 3]
```diff
@@ -69,14 +70,15 @@ Example schema change:
...
```
Comment (lolopinto) [inline README.md:71]: `user_schema.ts`

File: README.md [hunk 4]
```diff
@@ -95,12 +97,15 @@ Example schema change:
...
```
Comment (lolopinto) [inline README.md:98]: `user_schema.ts`

File: README.md [hunk 5]
```diff
@@ -95,12 +97,15 @@ Example schema change:
...
 // this adds a User object with a firstName and lastName
-import { BaseEntSchema, Field, StringType } from "@snowtop/ent";
-
-export default class User extends BaseEntSchema {
-  fields: Field[] = [
-    StringType({ name: "FirstName" }),
-    StringType({ name: "LastName" }),
-  ];
-}
+// src/schema/user.ts
```
Comment (lolopinto) [inline README.md:100]: `user_schema.ts`

File: package.json [hunk 6]
```diff
@@ -27,10 +27,11 @@
   },
   "homepage": "https://github.com/lolopinto/ent-starter#readme",
   "dependencies": {
-    "@snowtop/ent": "^0.0.34",
+    "@snowtop/ent": "^0.1.0-alpha34",
```
Comment (lolopinto) [inline package.json:30]: `^0.1.0-alpha93`

File: develop.Dockerfile [hunk 7]
```diff
@@ -1,4 +1,5 @@
-FROM ghcr.io/lolopinto/ent:0.0.35-nodejs-17-dev
+FROM ghcr.io/lolopinto/ent:0.1.0-alpha.5-nodejs-17-dev
```
Comment (lolopinto) [inline develop.Dockerfile:1]: `ghcr.io/lolopinto/ent:0.1.0-alpha.30-nodejs-17-dev`

**Review by lolopinto — CHANGES_REQUESTED** [review]
some changes

**Comment by mrtom** [comment]
OK, hopefully all fixed. I created a new repo from my copy of the template and checked I could run through the README with pg and spin up the GQL server without issue too.

**Review by lolopinto — APPROVED** [review]

**Comment by lolopinto** [comment]
thanks!

**Pull request merged**